package/base-files/files/lib/functions/uci-defaults.sh

```diff
@@ -771,6 +771,17 @@ ucidef_add_wlan() {
 	ucidef_wlan_idx="$((ucidef_wlan_idx + 1))"
 }
 
+network_device_port_range() {
```

Comment:
I think it would be better to add a function like
-> ucidef_set_interface_netdev_range wan eth 1 32
this will avoid invoking a sub shell in the last patch of this series

Reply:
I just added the `ucidef_set_interface_netdev_range` function. I also removed the subshell for the `seq` call inside the function.